viz/cmd/authz.go

```diff
@@ -73,8 +73,10 @@ func NewCmdAuthz() *cobra.Command {
 			}
 
 			cols := []table.Column{
+				table.NewColumn("ROUTE").WithLeftAlign(),
 				table.NewColumn("SERVER").WithLeftAlign(),
-				table.NewColumn("AUTHZ").WithLeftAlign(),
+				table.NewColumn("AUTHORIZATION_POLICY").WithLeftAlign(),
+				table.NewColumn("SERVER_AUTHORIZATION").WithLeftAlign(),
```

Comment:
previously, we abbreviated "authorization" to "authz" here...now, we've added more columns, and also expanded the abbreviation to the full word "authorization"...in general, i'm in favor of using the whole word instead of an abbreviation, but how well does the table fit in smaller terminals now? should we continue using the abbreviation so that it's fewer characters wide?

Reply:
In the example above, the header line is 130 characters wide and would shrink down to 118 characters wide with this abbreviation.  I'm not sure what is reasonable to expect for terminal width, given that we're well past 80 anyway.  I'm open to it if we think it would help.